sdcm/logcollector.py

```diff
@@ -1222,6 +1222,35 @@ def get_gce_instances_by_testid(self):
                                                       instance=instance,
                                                       global_ip=instance.public_ips[0],
                                                       tags={**self.tags, "NodeType": "loader", }))
+        if self.params["use_cloud_manager"]:
+            try:
+                from cluster_cloud import \
+                    get_gce_manager_instance_by_cluster_id  # pylint: disable=import-outside-toplevel
+            except ImportError:
+                LOGGER.error("Couldn't collect Siren manager logs, cluster_cloud module isn't installed")
+            else:
+                cloud_manager_id = self.params["cloud_manager_id"] if "cloud_manager_id" in self.params else None
+                LOGGER.info("GCE manager instance id: %s", cloud_manager_id)
+                if cloud_manager_id:
+                    cloud_cluster_id = self.params["cloud_cluster_id"]
+                    try:
+                        instance = get_gce_manager_instance_by_cluster_id(cluster_id=cloud_cluster_id)
+                        if not instance:
+                            raise ValueError("Cloud manager for the cluster %s not found", cloud_cluster_id)
+                    except Exception as exc:
+                        LOGGER.error("Failed to get cloud manager instance. Error: %s", str(exc))
```

Comment:
No need to explicitly convert the exception to a string: `%s` in the format string do the same.

Reply:
fixed